Uplift/Models/Gym.swift

```diff
@@ -92,7 +92,12 @@ struct Gym {
             return DailyGymHours(gymHoursDataId: gymHoursDataId)
         })
 
-        facilities = []
+        facilities = gymData.facilities.compactMap {
+            if let facility = $0 {
+                return Facility(facilityData: facility)
+            }
+            return nil
```

Comment:
lets use a `guard let` here just to be consistent with the other `compactMap`s in the PR